app/javascript/src/utils/api/properties.js

```diff
@@ -6,14 +6,19 @@ export const getProperties = async (params) => {
   const { data } = await axios.get(
     `${ENDPOINT_API}/properties/?q=${params}`
   )
-
```

Comment:
I'm a fan of having the return statement on its own line so it reads clearer. But that can be down to a vote amongst us devs